packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4285,6 +4288,18 @@ export default class Meeting extends StatelessWebexPlugin {
           EVENT_TRIGGERS.MEETING_RECONNECTION_SUCCESS
         );
         LoggerProxy.logger.log('Meeting:index#reconnect --> Meeting reconnect success');
+
+        // @ts-ignore
+        this.webex.internal.newMetrics.submitClientEvent({
+          name: 'client.media.recovered',
+          payload: {
+            recoveredBy: 'new',
+          },
+          options: {
+            meetingId: this.id,
+          },
+        });
+        this.reconnectionManager.setStatus(RECONNECTION.STATE.COMPLETE);
```

Comment:
It doesn't feel right that we have to do this from here, it means that basically the ReconnectionManager is not really fully managing the reconnection: the private details of it (status) are leaked outside and the logic of what we have to do as part of reconnection (wait for answer, wait for "connected" state) is partially here in Meeting and not all inside reconnection manager. Also the CA event for starting reconnection (client.media.reconnecting) is sent from ReconnectionManager, while the final event (client.media.recovered) is sent from Meeting. It feels like to fix this we would need a slightly bigger refactor, so maybe this is fine for now and we'll do it as part of the call-end CA events work in Q3

Reply:
We've discussed this offline but for anyone reading this PR we have agreed that although it is not ideal to have reconnection logic split between classes like this it would require a greater effort to refactor to allow the reconnection to be completely contained within ReconnectionManager. 

I've created https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-492161 under the Q3 call-end epic so that we don't forget that this is something we would like to do at some point, but we will prioritise that work when we start work on call-end stuff.